pkg/test/resourcefixture/testdata/basic/secretmanager/v1beta1/fullsecretmanagersecret-manual-direct/_generated_object_fullsecretmanagersecret-manual-direct.golden.yaml

```diff
@@ -2,8 +2,6 @@ apiVersion: secretmanager.cnrm.cloud.google.com/v1beta1
 kind: SecretManagerSecret
 metadata:
   annotations:
-    alpha.cnrm.cloud.google.com/reconciler: direct
-    cnrm.cloud.google.com/management-conflict-prevention-policy: none
```

Comment:
What do we want to do about management-conflict-prevention-policy ?  cc @cheftako